src/it/scala/com/wavesplatform/it/BalancesSpecification.scala

```diff
@@ -0,0 +1,318 @@
+package com.wavesplatform.it
+
+import java.io.IOException
+
+import com.wavesplatform.settings.Constants
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{BeforeAndAfterAll, FunSuite, Matchers, RecoverMethods}
+import scorex.transaction.TransactionParser.TransactionType
+
+import scala.concurrent.Await._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.Random
+
+class BalancesSpecification(allNodes: Seq[Node]) extends FunSuite with Matchers with ScalaFutures
+  with IntegrationPatience with BeforeAndAfterAll with RecoverMethods {
+  val sender = Random.shuffle(allNodes).head
+  val richBalance = sender.address
+
+  var first: String = _
+  var second: String = _
+  var third: String = _
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+
+    first = result(sender.createAddress, 5.seconds)
```

Comment:
Well, that's actually debatable. Result, well, provides a result, and `5 seconds` is also a hint. What I'd like to see is a comment that you create a new address here to simplify balance checks in the tests.